Person-CRUD/Person.Business/IPersonManager.cs

```diff
@@ -0,0 +1,18 @@
+﻿using Person_CRUD.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Bussiness
+{
+    public interface IPersonManager
+    {
+        public Task<bool> CreatePerson(Person person);
```

Comment:
no need to put any access modifier here as the Interface comes up Public by default.